privateer.json

```diff
@@ -0,0 +1,35 @@
+{
+    "servers": [
+        {
+            "name": "annex2",
+            "hostname": "annex2.montagu.dide.ic.ac.uk",
+            "port": 10023,
+            "container": "privateer_server_wodin_epimodels"
+        }
+    ],
+    "clients": [
+        {
+            "name": "epimodels",
+            "backup": ["redis-data"],
+            "schedule": {
+                "port": 9090,
+                "jobs": [
+                    {
+                        "server": "annex2",
+                        "volume": "redis-data",
+                        "schedule": "@hourly"
```

Comment:
oof every hour

Reply:
yeah, it moves so little data this takes a second or so